032-MLOpsFromScratch/Coach/01-TimeSeriesForecasting.md

```diff
@@ -0,0 +1,43 @@
+# Challenge 1 – Start Project and Import Repo into Azure DevOps
+
+[< Previous Challenge](./00-prereqs.md) - **[Home](../README.md)** - [Next Challenge >](./02-UnitTesting.md)
+
+## Solution
+
+1.  Create and setup a new project in Azure DevOps
+
+    1.  Download and extract `Data_and_Code.zip` from Teams Channel 
+
+    2.  [Create new service connections](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml) in Project Settings for your Azure ML service and Azure Subscription using service

```

Comment:
1 and 2 feel redundant from the student challenge text?

Remember, don't duplicate or it will make maintenance a nightmare later on.

Reply:
Made changes here too. Removed the duplicate part.